packages/trader/src/Modules/CFD/Helpers/constants.js

```diff
@@ -29,6 +40,10 @@ const getPlatformMt5DownloadLink = (platform = undefined) => {
     }
 };
 
+const getDXTradeWebTerminalLink = category => {
+    return category === 'real' ? 'https://dx.deriv.com' : 'https://dx-demo.deriv.com/';
+};
```

Comment:
IMO, it is a good pattern to import these constant URLs from a separate config file that holds these constants instead of hardcoding them directly into code. by the way, it is not a big deal

```suggestion
const getDXTradeWebTerminalLink = category => {
    return category === 'real' ? REAL_DXTRADE_URL : DEMO_DXTRADE_URL;
};
```